tests/unit/Rules/CnhTest.php

```diff
@@ -98,30 +98,30 @@ public function validCnhProvider()
     public function invalidCnhProvider()
     {
         return [
-               ['12651316461'],
-               ['14397322871'],
-               ['14375711312'],
-               ['12996843266'],
-               ['14375711511'],
-               ['12615113411'],
-               ['13247161316'],
-               ['11258751259'],
-               ['11739751581'],
-               ['13375637514'],
-               ['12542551342'],
-               ['11718111411'],
-               ['11836511948'],
-               ['14365445978'],
-               ['14324384312'],
-               ['14339482949'],
-               ['11136521151'],
-               ['11612581127'],
-               ['11613454741'],
-               ['14129251992'],
-               ['13411741211'],
-               ['13417248311'],
-               ['11671431345'],
-               ['13292694415'],
```

Comment:
Thanks for contributing! 

First of all, I'm sorry by my late. Secondly, I didn't get exactly why the current algorithm is not correct, can you elaborate?

Also, the numbers removed from here, should all them be valid? If so, they must be put on the `validCnhProvider()` but I need some clarification about why.